src/EditorFeatures/Next/NavigableSymbols/NavigableSymbolService.Source.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Editor.GoToDefinition;
+using Microsoft.CodeAnalysis.LanguageServices;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.VisualStudio.Language.Intellisense;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.CodeAnalysis.Editor.Host;
+
+namespace Microsoft.CodeAnalysis.Editor.NavigableSymbols
+{
+    internal partial class NavigableSymbolService : INavigableSymbolSourceProvider
+    {
+        private partial class Source : INavigableSymbolSource
```

Comment:
Names like this make it hard to use navigate-to.  Can you just call it the NavigableSymbolSource?